games/g.yaml

```diff
@@ -259,6 +259,27 @@
   added: 2025-08-26
   updated: 2025-08-26
 
+- name: Glider for Apple II
```

Comment:
```suggestion
- name: a2tools
```
should be named after the monorepo; multiple entries can be added under originals

Reply:
my bad I think in this case because the URL is different it can be added like this